packages/angular/migrations.json

```diff
@@ -1514,6 +1514,19 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "14.7.0": {
+      "version": "14.7.0-beta.0",
```

Comment:
There's a `14.7.0-beta.1` version already released. Let's bump the target version here and in the other `migrations.json` files included in this PR.

Reply:
good spot